### ISSUE TITLE
FIX missing div fold-icon in Elements tab

### ISSUE THREAD

**Issue by jaykobo** [taniarascia/new-moon-chrome-devtools#6]
I really **love** your code coloring! But unfortunately I miss the fold-icon (inside Elements tab) next to a div to open/close it - which makes the plugin unusable for me. 
For better understanding I've attached two screenshots.

### Fold-icon (don't know how to name it correcty):
![dev-tools-without-default](https://user-images.githubusercontent.com/24397393/66898430-7af0b000-eff9-11e9-9497-2376f9c2fd01.png)

### Missing Icons in Elements tab:
![dev-tools-without-fold](https://user-images.githubusercontent.com/24397393/66898431-7af0b000-eff9-11e9-9a0a-00b09d3742ec.png)

### System Informations
- MacBook Pro (Retina, 15')
- macOS Mojave, version 10.14.6
- Google Chrome Version 77.0.3865.120

**Comment by taniarascia** [comment]
You're right! I was wondering what happened to it. I don't know why it disappeared when I didn't change anything.

**Comment by taniarascia** [comment]
Maybe related to https://support.google.com/chrome/thread/2670873?hl=en.

Sometimes I see them, sometimes I don't. Either way, the function is still there, just the icon is hidden :-/

**Comment by jaykobo** [comment]
I have tried other themes and the icons remain visible. And without installed themes the error never occurs with me. But maybe it has something to do with your linked bug.

Thank you for the answer.

**Comment by taniarascia** [comment]
Sorry, I don't think that bug is related. I'm not sure what's causing this issue when I didn't change anything from it working to not working, but I would like to get around to fixing it. I'm just clicking the empty space for now...)
